packages/live-share-inking/src/canvas/DryWetCanvas.ts

```diff
@@ -0,0 +1,269 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the Microsoft Live Share SDK License.
+ */
+
+import { InkingCanvas } from "./InkingCanvas";
+import { getPressureAdjustedSize, IPointerPoint, DefaultPenBrush, IBrush, toCssColor } from "../core";
+import { computeQuadBetweenTwoCircles, computeQuadBetweenTwoRectangles, IQuadPathItem } from "../core/Internals";
+
+/**
+ * Represents the base class from wet and dry canvases, implementing the common rendering logic.
+ */
+export abstract class DryWetCanvas extends InkingCanvas {
```

Comment:
Perhaps split classes into separate files?

Reply:
I personally find it overkill... but just insist once and I'll do it :)